app/views/physical_storage/edit.html.haml

```diff
@@ -1,3 +1,2 @@
 = render :partial => "layouts/flash_msg"
 = react 'PhysicalStorageForm', :recordId => @storage.id.to_s
-
```

Comment:
@GilbertCherrie Could you add these empty lines back?

Reply:
yeah